source/manual/govuk-env-sync.html.md

```diff
@@ -120,3 +120,7 @@ Wherein we are
 
 ### Icinga checks
 For every `govuk_env_sync::task:`, a passive Icinga alert `GOV.UK environment sync <title>` is created. They are updated on exit of the sync script.
+
+If you get an Icinga alert about a failing job, investigate whether there were any new problems as it does occasionally fail. If not, it can be acked for 24 hours and left to run overnight again.
+
+A failure may mean that links to apps in Signon in staging link to production apps. Running the [data_sync_complete](https://deploy.staging.publishing.service.gov.uk/job/Data_Sync_Complete/) job may fix this.
```

Comment:
This line is probably better suited to this page https://docs.publishing.service.gov.uk/manual/alerts/data-sync.html because it's not related to the new `govuk_env_sync` system.

Reply:
Ah right, I've updated that now